recipes/beavers/meta.yaml

```
@@ -0,0 +1,49 @@
+{% set name = "beavers" %}
+{% set version = "0.13.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/beavers-{{ version }}.tar.gz
+  sha256: cb1dfa6cbfb110e2c00a9f9682b03299e1a1737c7c4ef102b345c5791dd04c82
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python {{ python_min }}
+    - poetry-core
+    - poetry-dynamic-versioning
+    - pip
+  run:
+    - python >={{ python_min }}
+    - confluent-kafka >=2.1.1
```

```suggestion
    - python-confluent-kafka >=2.1.1
```

On conda-forge, this name appears to be different.